src/cache_cf.cc

```diff
@@ -298,139 +299,220 @@ skip_ws(const char* s)
     return s;
 }
 
+/// Parsers included configuration files identified by their filenames or glob
+/// patterns and included at the given nesting level (a.k.a. depth).
+/// For example, parses include files in `include /path/to/include/files/*.acl`.
+/// \returns the number of errors (that did not immediately terminate parsing)
 static int
-parseManyConfigFiles(char* files, int depth)
+parseIncludedConfigFiles(const SBuf &paths, const int depth)
 {
     int error_count = 0;
-    char* saveptr = nullptr;
+    Parser::Tokenizer tk(paths);
 #if HAVE_GLOB
-    char *path;
     glob_t globbuf;
     int i;
     memset(&globbuf, 0, sizeof(globbuf));
-    for (path = strwordtok(files, &saveptr); path; path = strwordtok(nullptr, &saveptr)) {
-        if (glob(path, globbuf.gl_pathc ? GLOB_APPEND : 0, nullptr, &globbuf) != 0) {
-            int xerrno = errno;
-            fatalf("Unable to find configuration file: %s: %s", path, xstrerr(xerrno));
+    while (auto path = NextWordWhileRemovingDoubleQuotesAndBackslashesInsideThem(tk)) {
+        if (glob(path->c_str(), globbuf.gl_pathc ? GLOB_APPEND : 0, nullptr, &globbuf) != 0) {
+            const auto xerrno = errno;
+            throw TextException(ToSBuf("Cannot find included configuration file named ", *path, ": ", xstrerr(xerrno)), Here());
```

Comment:
This is a UI change. Please document in the PR description, or revert to the original FATAL message text.

Reply:
> Please document in the PR description, or revert to the original FATAL message text.

Commit https://github.com/squid-cache/squid/commit/d0e01dd0679a6959155eb0f6045444052a9026a2 restores the original message text.